Dockerfile

```diff
@@ -0,0 +1,45 @@
+# Start from a Debian image with the latest version of Go installed
+# and a workspace (GOPATH) configured at /go.
+FROM marcbachmann/libvips:latest
```

Comment:
I don't know, happy to try it.

Agreed on the TODO. Happy to do that early next week.